aspnetcore/migration/20_21.md

```diff
@@ -41,7 +41,7 @@ Update the project file:
   * Microsoft.Extensions.Caching.SqlConfig.Tools  (`dotnet sql-cache`)
   * Microsoft.Extensions.SecretManager.Tools (`dotnet user-secrets`)
 * Optional: you can remove the **&lt;DotNetCliToolReference&gt;** element for `Microsoft.VisualStudio.Web.CodeGeneration.Tools`. You can replace this tool with a globally installed version by running `dotnet tool install -g dotnet-aspnet-codegenerator`.
-* For 2.1, a [Razor Class Library](xref:razor-pages/ui-class) is the recommended solution to distribute Razor files. If your app uses embedded views, or otherwise relies on runtime compilation of Razor files, add `<CopyRefAssembliesToPublishDirectory>true</CopyRefAssembliesToPublishDirectory>` to your project file.
+* For 2.1, a [Razor Class Library](xref:razor-pages/ui-class) is the recommended solution to distribute Razor files. If your app uses embedded views, or otherwise relies on runtime compilation of Razor files, add `<CopyRefAssembliesToPublishDirectory>true</CopyRefAssembliesToPublishDirectory>` to a `<PropertyGroup>` in your project file.
```

Comment:
Does it not matter which PropertyGroup it goes in?

Reply:
Correct ... doesn't matter. We should only have one in our samples/templates, so I don't think it will be confusing if we don't elaborate. For props used within the file, they're evaluated down the project file in order, so it's also 'last one wins.'